ui/hooks/src/use-close-notification.ts

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+export function useCloseNotification(key: string, initialValue = false) {
+  const [closed, setClosed] = useState(() => {
+    if (window?.localStorage) {
+      try {
+        const closedFlag = window.localStorage.getItem(key);
+        if (closedFlag) {
+          return JSON.parse(closedFlag);
+        } else {
+          window.localStorage.setItem(key, JSON.stringify(initialValue));
+          return initialValue;
+        }
+      } catch (err) {
+        // console.log(err);
+        return initialValue;
+      }
+    }
+  });
+
+  const setValue = value => {
```

Comment:
can we add a type to the value parameter?

Reply:
The value here should be `React.Dispatch<React.SetStateAction<boolean>>` which is not something i'm expecting to see in a simple setter function. Better stick to just `boolean` and remove extra logic that is not needed at the moment

Reply:
I have updated the function accordingly.